### ISSUE TITLE
New request: Plants for a Future

### ISSUE THREAD

**Issue by Samandamand** [openzim/zim-requests#745]
- Website URL: https://PFAF.org
- License: Creative Commons
- Desired ZIM Title: PFAF
- Desired ZIM Description: free-to-use information database for those interested in edible and useful plants.
- Desired ZIM Icon –png (URL or attach one): https://www.plantetp.eu/wp-content/uploads/2022/01/plants-for-the-future-e1679909954788.jpg
- Language (ISO 639-3): Eng
- Is this a MediaWiki?: No?

**Comment by RavanJAltaie** [comment]
Recipe created
https://farm.openzim.org/recipes/pfaf.org_en_all
Will update the library link once ready

**Comment by Samandamand** [comment]
Will this be attempted again with a "Page Extra Delay" of 10 seconds like suggested in the above thread or was it already attempted?

**Comment by Samandamand** [comment]
I don't know if this is any help but this person seems to have scraped the data from PFAF.org https://github.com/saulshanabrook/pfaf-data

**Comment by benoit74** [comment]
Yes, this is a great news because this means we could definitely create a custom scraper. But as you know, this is a significant effort. 

I suggest to wait for zimit2 and check what happens with this new scraper version, maybe it will work way better regarding Cloudflare (it proved to in some cases).

If it doesn't work better, then we will tag this issue "scraper needed" and wait for funding / an external contributor / ...

**Comment by Samandamand** [comment]
Is it possible to zim the data from here instead since they already did the scraping? 
 https://lite.datasette.io/?url=https://saulshanabrook.github.io/pfaf-data/data.sqlite#/data/plant_data

**Comment by benoit74** [comment]
Indeed, a custom scraper could benefit from using https://saulshanabrook.github.io/pfaf-data/ dataset

**Comment by benoit74** [comment]
But we cannot ZIM the URL you've provided with Zimit, it is not going to work unfortunately. Maybe setting-up a simple website working only client-side with the sqlite database of https://saulshanabrook.github.io/pfaf-data/ and then running zimit on it would a quick win.
